packages/kit/src/runtime/client/singletons.js

```diff
@@ -1,7 +1,14 @@
 /** @type {import('./router').Router?} */
 export let router;
 
-/** @param {import('./router').Router?} _ */
-export function init(_) {
-	router = _;
+/** @type {import('./renderer').Renderer} */
+export let renderer;
+
+/** @param {{
+ *   router: import('./router').Router?;
+ *   renderer: import('./renderer').Renderer;
+ * }} opts */
```

Comment:
VS Code's syntax highlighting won't work if you put the `@param` on the same line as the `/**` unless it's just a single-line comment
```suggestion
/**
 * @param {{
 *   router: import('./router').Router?;
 *   renderer: import('./renderer').Renderer;
 * }} opts
 */
```